### DISCUSSION TITLE
Using CI_JOB_TOKEN at Gitlab

### DISCUSSION THREAD

**Discussion by roylisto** [goreleaser/goreleaser#2384] [Q&A] [answered]
Hi I get an error when using CI_JOB_TOKEN 
`release failed after 175.20s error=github/gitlab/gitea releases: failed to publish artifacts: GET https://my.company.com/api/v4/projects/my-project/app-cli/releases/v0.1.0: 401 {message: 401 Unauthorized}`

does it mean CI_JOB_TOKEN cannot publish to the release page?

**Comment by caarlos0** [comment, marked as answer]
you need an api token, please check the docs at:

- https://goreleaser.com/scm/gitlab/
- https://goreleaser.com/ci/gitlab/

**Reply by roylisto** [reply]
so I need to create another GitLab account and use that account to create API token?

**Reply by caarlos0** [reply]
Yup
